contrib/completions/bash/openshift

```diff
@@ -5357,6 +5425,7 @@ _openshift_admin()
     commands+=("create-bootstrap-policy-file")
     commands+=("create-login-template")
     commands+=("create-provider-selection-template")
+    commands+=("create-error-template")
```

Comment:
create-login-error-template or create-oauth-error-template? unless you think the error template will be used for something other than the oauth flow in the future